src/main/java/mate/academy/bookstore/model/Book.java

```diff
@@ -25,6 +25,5 @@ public class Book {
     @Column(nullable = false)
     private BigDecimal price;
     private String description;
-    @Column(name = "cover_image")
```

Comment:
why did you delete the column name?

Reply:
I was told exactly the same column name would be parsed by sql without any annotations. Isn't it redundant code then?

Reply:
It was my comment that it is not required, because Hibernate does it by default, but it's if you do it explicitly, so it's for you to decide, there is no right answer